src/Controllers/ApplicationFormController.php

```diff
@@ -42,6 +42,7 @@ public function __invoke(Request $request, Response $response, array $args)
                     'data' => [
                         'cohorts' => $this->applicationFormModel->getCohorts(),
                         'hearAbout' => $this->applicationFormModel->getHearAbout(),
+                        'stages' => $this->applicationFormModel->getStages(),
```

Comment:
should this be on the stage model?

Reply:
should this use the $this->stageModel->getStageTitles() ?

Reply:
No longer necessary so we took this out.